### ISSUE TITLE
Show Warning Color at 70% of Threshold

### ISSUE THREAD

**Issue by daniellmb** [kentcdodds/ng-stats#42]
While we are contemplating refactoring colors, how about calling attention to the stats when the metric (digest time or watch count) is within 30% of the limit.

Right now we just have green (everything is ok) and red (over the threshold limit). It could be helpful to call attention to the metrics when the application is getting close to the configured limit. Say at 70% of the threshold you should probably audit the application for potential issues.

Both `colorLog` and `addDataToCanvas` could call something like this to look up the color.

```js
function getColor(metric, threshold) {
    if (metric > threshold) {
        return 'red';
    } else if (metric > 0.7 * threshold) {
        return 'orange';
    }
    return 'green';
}
```
```js
// color log ...
if (thingToLog === 'digest') {
  color = 'color:' + getColor(tracked, opts.digestTimeThreshold);
} else if (thingToLog === 'watches') {
  color = 'color:' + getColor(tracked, opts.watchCountThreshold);
}
// ...
```

Thoughts? I'd be happy to submit a PR :)

**Comment by kentcdodds** [comment]
Sounds awesome!

**Comment by daniellmb** [comment]
Ok, I'll combine #41 and #42 in a single PR

**Comment by kentcdodds** [comment]
:+1:

**Comment by kentcdodds** [comment]
Released in `2.4.0`. Thanks!
